lib/internal/modules/package_json_reader.js

```diff
@@ -11,7 +11,10 @@ const {
 const { internalModuleReadJSON } = internalBinding('fs');
 const { toNamespacedPath } = require('path');
 const { kEmptyObject } = require('internal/util');
-const { fileURLToPath, pathToFileURL } = require('internal/url');
+
+// It is important to require from 'url' and not from 'internal/url'
+// in order to add url module to bootstrapped modules for workers.
```

Comment:
I don't know the answer to this question, but is it necessary to have `url` bootstrapped? The point of `test-bootstrap-modules` is not so much that those modules have to be bootstrapped, but that it points out when the modules being bootstrapped changes and forces a conscious decision that the change was intentional (i.e. then update the test) or not (i.e. update the code changes in the pull request).

Reply:
For non-workers, we already bootstrap `url`. So, I believe the question is: should we bootstrap URL for non-worker, and not for workers?